CONTRIBUTING.md

```diff
@@ -45,41 +60,61 @@ by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
     git checkout -b feature/myfeature
     ```
     It is best to merge one's changes *as fast as possible* (i.e. do not wait for 6 months) to avoid merging conflicts
-3. Open [Protégé](https://protege.stanford.edu/) and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers).
+3. 📙 or 📗 Open [Protégé](https://protege.stanford.edu/) or a text editor and work on the ontology. If you haven't already, make sure you change your protégé settings to use [numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers).
 
     One can also edit the `oeo.omn` file in a text editor
-4. Before committing your changes, open the `oeo.omn` file with Protégé and save the file from Protégé. You should also check if you included inconsistencies by following [this ontology test procedure](https://github.com/OpenEnergyPlatform/ontology/wiki/ontology-test-guide)
+4. 📙 Before committing your changes, open the `oeo.omn` file with Protégé and save the file from Protégé. You should also check if you included inconsistencies by following [this ontology test procedure](https://github.com/OpenEnergyPlatform/ontology/wiki/ontology-test-guide)
 
     See the "Conventions" section below for commit messages format tips
-5. Push your local branch on the remote server `origin`
+5. 🔷 Get your changes online
+    
+    stage the files you changed
+    ```bash
+    git add [file_name]
+    ```
+
+    commit your changes
+    ```bash
+    git commit [-m " <commit_message> "]
+    ```
+
+    push your branch to the remote server
+
     ```bash
     git push
     ```
     If your branch does not exist on the remote server yet, git will provide you with instructions, simply follow them
-6. Make sure that all automated tests are successful. This will be indicated by a green or red icon next to your most recent commit.
-7. Submit a pull request (PR)
+6. 🍏 Make sure that all automated tests are successful. This will be indicated by a green or red icon next to your most recent commit.
```

Comment:
```suggestion
6. 🍏 Make sure that all automated tests are successful. This will be indicated by a green icon next to your most recent commit. What to do in case an error occured, is described in section "Fixing merge conflicts".```

Reply:
The errors here are created by the changes itself, not merge conflicts. So I don't think this will help

Reply:
Ok. But we should comment on what to do in such a case. Write a comment in the PR and ask for help from the ontology-expert-team?

Reply:
yes, good idea